Sources/Plasma/Apps/plClient/plClient.cpp

```diff
@@ -517,8 +517,8 @@ bool plClient::InitPipeline()
 
     float   yon = 500.0f;
 
-    pipe->SetFOV( 60.f, int32_t( 60.f * pipe->Height() / pipe->Width() ) );
-    pipe->SetDepth( 0.3f, yon );
+    pipe->SetFOV(60.f, 60.f * (pipe->Height() / pipe->Width()));
```

Comment:
this is `float * (int / int)`, which will cause truncation in the divide.  It therefore needs to be either without parens, or with parens around the multiplication.